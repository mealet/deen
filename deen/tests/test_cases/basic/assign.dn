fn main() i32 {
  // integer
  let int: i32 = 1;
  println!("{}", int);

  int = 2;
  println!("{}", int);

  int = 3;
  println!("{}", int);

  // floating
  let float: f64 = 1.0;
  println!("{}", float);

  float = 1.23;
  println!("{}", float);

  float = 1.33;
  println!("{}", float);

  // string
  let string = "hello";
  println!("{}", string);

  string = "world";
  println!("{}", string);

  string = "test";
  println!("{}", string);

  // char
  let character = 'a';
  println!("{}", character);

  character = 'b';
  println!("{}", character);

  character = 'c';
  println!("{}", character);

  return 0;
}
