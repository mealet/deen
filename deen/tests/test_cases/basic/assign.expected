1
2
3
1.000000
1.230000
1.330000
hello
world
test
a
b
c
